src/puppetlabs/puppetdb/command.clj

```diff
@@ -270,26 +266,25 @@
 ;; Fact replacement
 
 (defn replace-facts*
-  [{:keys [payload annotations version]} db]
-  (let [{:keys [id]} annotations
-        {:keys [certname values] :as fact-data} payload
+  [{:keys [payload annotations version] :as command} db]
```

Comment:
I think version should be gone. I think command is useful as it says the kind of map that is passed in, as opposed to a map that has a payload an annotations. If it's more distracting than useful though, I can remove it.